### ISSUE TITLE
Craft 4: Plugin breaks GraphQL, getPublicVolumes() is removed

### ISSUE THREAD

**Issue by niektenhoopen** [ethercreative/seo#401]
### Description
GraphQL does not work (at all) when the SEO plugin is enabled

It gives the following error:

```
{
  "errors": [
    {
      "message": "Calling unknown method: craft\\services\\Volumes::getPublicVolumes()"
    }
  ]
}
```

`getPublicVolumes()` is removed in https://github.com/craftcms/cms/commit/e93834d6b7b3ee2a5a7b9cc8b22e6249d04ed18b

### Additional info

- Craft version: 4
- SEO version: 4.0.0
- PHP version: 8.0

**Comment by niektenhoopen** [comment]
Already fixed in 4.0.1

**Comment by niektenhoopen** [comment]
@Tam Seems that Packagist does not recognise 4.0.1 (yet): https://packagist.org/packages/ether/seo

**Comment by niektenhoopen** [comment]
Also, I have issues with the current "fix" in 4.0.1: 
<img width="555" alt="Screenshot 2022-05-14 at 15 24 45" src="https://user-images.githubusercontent.com/3450011/168427767-e7c0baa9-b067-46bb-8f6f-82de68fa9444.png">

**Comment by jesuismaxime** [comment]
Any plan to merge that PR ☝️  that would be appreciated!

**Comment by jesuismaxime** [comment]
Any release date (on the plugin store - or available through composer) for 4.0.2 ?

**Comment by niektenhoopen** [comment]
@jesuismaxime As a work-around, you could theoretically require: `"ether/seo": "v4.x-dev#9f56366"`
